src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3526,4 +3370,135 @@
+    <value>items</value>
+  </data>
+  <data name="StatusCenter_CompressCanceled_Header" xml:space="preserve">
+    <value>Canceled compressing "{0}" items</value>
+    <comment>Shown in StatusCenter card item handling compress operation.</comment>
+  </data>
+  <data name="StatusCenter_CompressComplete_Header" xml:space="preserve">
+    <value>Successfully {0} items compressed</value>
+    <comment>Shown in StatusCenter card item handling compress operation.</comment>
+  </data>
+  <data name="StatusCenter_CompressFailed_Header" xml:space="preserve">
+    <value>Failed to compress {0} items</value>
+    <comment>Shown in StatusCenter card item handling compress operation.</comment>
+  </data>
+  <data name="StatusCenter_CompressInProgress_Header" xml:space="preserve">
+    <value>Compressing {0} items</value>
+    <comment>Shown in StatusCenter card item handling compress operation.</comment>
+  </data>
+  <data name="StatusCenter_CopyCanceled_Header" xml:space="preserve">
+    <value>Canceled copying {0} items</value>
+    <comment>Shown in StatusCenter card item handling copy operation.</comment>
+  </data>
+  <data name="StatusCenter_CopyComplete_Header" xml:space="preserve">
+    <value>Successfully {0} items copied</value>
+    <comment>Shown in StatusCenter card item handling copy operation.</comment>
+  </data>
+  <data name="StatusCenter_CopyFailed_Header" xml:space="preserve">
+    <value>Failed to copy items</value>
+    <comment>Shown in StatusCenter card item handling copy operation.</comment>
+  </data>
+  <data name="StatusCenter_CopyInProgress_Header" xml:space="preserve">
+    <value>Copying {0} items</value>
+    <comment>Shown in StatusCenter card item handling copy operation.</comment>
+  </data>
+  <data name="StatusCenter_DecompressCanceled_Header" xml:space="preserve">
+    <value>Canceled decompressing "{0}"</value>
+    <comment>Shown in StatusCenter card item handling decompress operation.</comment>
+  </data>
+  <data name="StatusCenter_DecompressComplete_Header" xml:space="preserve">
+    <value>Successfully "{0}" extracted</value>
+    <comment>Shown in StatusCenter card item handling decompress operation.</comment>
+  </data>
+  <data name="StatusCenter_DecompressFailed_Header" xml:space="preserve">
+    <value>Failed to extract "{0}"</value>
+    <comment>Shown in StatusCenter card item handling decompress operation.</comment>
+  </data>
+  <data name="StatusCenter_DecompressInProgress_Header" xml:space="preserve">
+    <value>Extracting "{0}"</value>
+    <comment>Shown in StatusCenter card item handling decompress operation.</comment>
+  </data>
+  <data name="StatusCenter_DeleteCanceled_Header" xml:space="preserve">
+    <value>Canceled deleting {0} items</value>
+    <comment>Shown in StatusCenter card item handling deleting operation.</comment>
+  </data>
+  <data name="StatusCenter_DeleteComplete_Header" xml:space="preserve">
+    <value>Successfully {0} items permanently deleted</value>
```

Comment:
```suggestion
    <value>Permanently deleted {0} items </value>
```

Reply:
‘Successfully’ is not needed for quick notice and for consistency?

Reply:
I think the icon and background color helps for that.

Reply:
How about the latter one? No worries?

Reply:
I'm not concerned about that, but we can keep an eye out for feedback.

Reply:
@mdtauk what you think will be what I agree.